src/Replicate.Net/Models/Prediction.cs

```diff
@@ -27,7 +27,7 @@ public class Prediction
     /// </summary>
     public string Status { get; set; } = null!;
 
-    public IPredictionInput Input { get; set; } = null!;
+    public JObject Input { get; set; } = null!;
 
     public object? Output { get; set; }
```

Comment:
What do about Output?
Also change to JObject ?

Or change Input yo object?

Reply:
I think Input to object is better. JObject, object both working.